<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SEPreview</class>
 <widget class="QDialog" name="SEPreview">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>274</width>
    <height>274</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>274</width>
    <height>274</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>274</width>
    <height>274</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Strucuring element</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="lbSEPreview">
     <property name="minimumSize">
      <size>
       <width>256</width>
       <height>256</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
